packages/wxt/src/core/zip.ts

```diff
@@ -19,6 +19,7 @@ import { normalizePath } from './utils/paths';
  */
 export async function zip(config?: InlineConfig): Promise<string[]> {
   await registerWxt('build', config);
+  await wxt.hooks.callHook('zip:start', wxt);
```

Comment:
Let's move `zip:start` to after we perform the build, right before we start zipping things. We already have other hooks like `build:start` that fire around this time.

Reply:
True